src/js/components/homepage/Homepage.jsx

```diff
@@ -153,11 +154,27 @@ export default class Homepage extends React.Component {
         });
     }
 
+    skippedNav(e) {
+        // don't update the URL due to potential React Router conflicts
+        e.preventDefault();
+        // scroll to the main-content id
+        const yPos = document.querySelector('#main-content').getBoundingClientRect().top;
```

Comment:
Pressing enter or clicking this button to skip the navigation scrolls the page but doesn't update the focus, so keyboard users still have to tab through the menu